objectherkenning_openbare_ruimte/training_pipeline/components/train_model.py

```diff
@@ -24,6 +25,21 @@
 aml_experiment_settings = settings["aml_experiment_details"]
 
 
+def load_training_parameters(json_file: str):
+    """Load parameters from a JSON file, returning an empty dictionary if the file is empty."""
+    if os.stat(json_file).st_size == 0:  # Check if file is empty
```

Comment:
Instead of only checking for an empty file, perhaps we can also allow to omit this completely in the config file (so no file) as a default? That would make it a bit more flexible.